app/categories/constants.py

```diff
@@ -491,6 +499,9 @@ def init_children(category: Category) -> None:
         child.article_category_name = (
             child.article_category_name or category.article_category_name
         )
+        child.exit_page = (
```

Comment:
Can you do `child.exit_page = child.exit_page or category.exit_page` similar above snippets?

Reply:
> Hiya Said, the logic here is that if the child doesn't have exit page then don't set it via the child
I am not the snippet does that

The snippet `child.exit_page = child.exit_page if child.exit_page is not None else category.exit_page` reads

When `child.exit_page` is truthy then `child.exit_page` is equal to itself otherwise child.exit_page is equal to `category.exit_page`

Reply:
I see, it can be None, False or True